pyproject.toml

```diff
@@ -0,0 +1,39 @@
+# Testing tools configuration
+[tool.coverage.run]
+branch = true
+
+[tool.coverage.report]
+show_missing = true
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+log_cli_level = "INFO"
+
+# Formatting tools configuration
+[tool.black]
+line-length = 99
+target-version = ["py38"]
+
+# Linting tools configuration
+[tool.ruff]
```

Comment:
Most of our charms still use `flake8` due to certain plugins, I guess this will be reverted to `flake8` in the future?

Reply:
Yes, this is just the skeleton brought to life by `charmcraft`.